pkg/orchestrator/config/config.go

```diff
@@ -157,3 +158,22 @@ func extractOrchestratorDDUrl() (*url.URL, error) {
 	}
 	return URL, nil
 }
+
+// NewOrchestratorForwarder returns
```

Comment:
`returns an orchestratorForwarder if the agent is a cluster-check runner, nil otherwise`?